webapp/channels/src/sass/components/_announcement-bar.scss

```diff
@@ -87,10 +87,7 @@
         content: "\00a0 ";
     }
 
-    .content__icon,
-    i {
-        margin-right: 4px;
-        font-family: 'compass-icons';
+    .icon {
```

Comment:
`.icon` already sets the `font-family` and the margin isn't needed since the spacing is defined by padding on the span which follows this